examples/axual/33_application_access_grant_authorizations.tf

```diff
@@ -0,0 +1,18 @@
+resource "axual_application_access_grant_authorization" "dash_consume_logs_example" {
+  application_access_grant = axual_application_access_grant.dash_consume_from_logs_in_example.id
+  status = "Approved" # Change this to Revoked in order to revoke approval. That can only be done after this current state has been applied
+}
+
+output "dash_consume_logs_example_status" {
+  value = axual_application_access_grant_authorization.dash_consume_logs_example.status
+}
+
+resource "axual_application_access_grant_authorization" "scraper_consume_support_example" {
+  application_access_grant = axual_application_access_grant.log_scraper_consume_from_support_in_example.id
```

Comment:
Is the` axual_application_access_grant` available as a data source? If the data owner wants to create an `axual_application_access_grant_authorization` resource in a different Terraform state file, the `axual_application_access_grant` will have to be accessed via a data source. Is that supported?

Reply:
We have not yet implemented any of the data sources. Currently you would have to have the id of the grant in order to do that. We have a story for creating the data sources and I think it's best to do them all at once when we have the Interface for all resources fully registered.